config/routes.rb

```diff
@@ -1,13 +1,19 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do 
+  # get 'likes/create'
+  # get 'comments/new'
+  # get 'comments/create'
```

Comment:
- [ ] Kindly remove commented codes from your project. :)

Reply:
Already implemented it! Thanks for the feedback! :smile: